validator/stateless_block_validator.go

```diff
@@ -31,7 +31,7 @@ type StatelessBlockValidator struct {
 	streamer        TransactionStreamerInterface
 	blockchain      *core.BlockChain
 	db              ethdb.Database
-	das             das.DataAvailabilityService
+	Das             das.DataAvailabilityService
```

Comment:
I don't see this used publicly anywhere. Probably better to keep it private like it was